src/features/vote/ActiveRequestsForm.tsx

```diff
@@ -199,7 +233,7 @@ const ActiveRequestsForm: FC<Props> = ({
         <thead>
           <tr>
             <th>Requested Vote</th>
-            <th>Proposal Detail</th>
+            {/* <th>Proposal Detail</th> */}
```

Comment:
Why is this commented out?

If it's meant to stay commented, might be useful to add a note explaining why.

Reply:
Oh yes, sorry, this is because we are hiding the anc data column for now. I can add a comment np, or just delete it. It's a trivial thing to add back in.

Reply:
Up to you -- I think either an explanatory note or deleting SGTM!